app/src/auth/components/Session/index.jsx

```diff
@@ -0,0 +1,61 @@
+// @flow
+
+import React, { type Node } from 'react'
+
+import Context, { type Props as State } from '$auth/contexts/Session'
+import { isLocalStorageAvailable } from '$mp/utils/storage'
+
+const SESSION_TOKEN_KEY = 'session.token'
+
+type Props = {
+    children: Node,
+}
+
+class Session extends React.Component<Props, State> {
```

Comment:
`SessionProvider` might be a more appropriate name?

Reply:
Yep, I'll rename!